src/mscorlib/src/System/Globalization/CalendarData.Windows.cs

```diff
@@ -123,13 +129,17 @@ internal static int GetCalendars(String localeName, bool useUserOverride, Calend
         {
             EnumCalendarsData data = new EnumCalendarsData();
             data.userOverride = 0;
+#if INSIDE_CLR
```

Comment:
I checked corert and not using CORECLR as expected. so I'll change my ifdef's to CORECLR.